### PR TITLE
[stdlib] Convert methods to computed properties when applicable

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

File: stdlib/String.tr [hunk 1]
```diff
@@ -3,6 +3,7 @@ type String {
     init(cString: *Int8) {
         self._storage = ByteArray(cString)
         self._storage.append(0)
+        println(self._storage)
```
Comment (segiddins) [inline stdlib/String.tr]: debugging

Comment (harlanhaskins) [reply stdlib/String.tr]: Thanks!

**Comment by segiddins** [comment]
:D this is wonderful

**Comment by harlanhaskins** [comment]
Good to merge?

**Pull request merged**